tasks/Divergent Association Task.md

```diff
@@ -0,0 +1,54 @@
+# Summary
+Measuring creativity
+Participants are asked to come up with 10 nouns that are as different as possible.
+
+# References
+Olson, Jay A., et al. “Naming Unrelated Words Predicts Creativity.” PNAS, National Academy of Sciences, 22 June 2021, www.pnas.org/content/118/25/e2022340118. 
+
+Link: https://www.pnas.org/content/118/25/e2022340118
+
+# Stimuli
+## The visual components
+N/A
+
+## Materials for alternative versions of the experiment 
+N/A
+
+# Procedure
+## Experiment 1
+The group of participants was selected entirely from undergraduate psychology courses from Melbourne, Australia, and from social media advertisements in Montreal, Canada.  The group was mostly female, between 18 and 20.
+
+This group follows the steps outlined.
```

Comment:
1. Participants were asked to generate 10 unrelated nouns in four minutes. Participants were given instructions and asked to write down the words with a Pen and Pencil on the same instructions sheet. 
The task had the following additional instructions.
> 1) Use only single words. We used this rule because computational methods can score single words with less ambiguity than phrases. Words such as “cul de sac” were accepted and automatically hyphenated.
> 2) Use only nouns (e.g., things, objects, concepts). This rule keeps the class of words similar since the distance between words varies based on their part of speech, such as whether they are nouns or adjectives.
> 3) Avoid proper nouns (e.g., no specific people or places).
> 4) Avoid specialized vocabulary (e.g., no technical terms). This rule and the previous one prevent participants from using words that are too specific, which is one strategy to artificially inflate the score. To enforce these rules, only lowercase words from a common dictionary (53) were used in the calculation.
> 5) Think of the words on your own (e.g., do not just look at objects in your surroundings). During pilot testing, many participants would look around their environment for inspiration when naming the words. This strategy resulted in lower scores since common objects on one’s desk are often semantically similar.
> 6) You will have 4 min to complete this task. In our initial testing (54), this amount of time was sufficient to complete the task without much time pressure.
2. . After the task, participants were asked what strategy they used, if any, to choose the words. 
> In Study 1A, two raters coded the responses based on 1) whether the 141 participants appeared to correctly follow the instructions and 2) whether they reported implementing a strategy such as naming the objects around them. Disagreements were resolved by discussion, and raters were liberal with their exclusions.